packages/flutter_webrtc/analysis_options.yaml

```diff
@@ -0,0 +1,50 @@
+include: package:lints/recommended.yaml
```

Comment:
The `flutter_webrtc_tizen` package contains no Dart code. This file (`analysis_options.yaml`) can be removed.

Reply:
Ok

Reply:
I suspect you misunderstood my comment?

Reply:
I have removed analysis_options.yaml file